cocos/ui/UIListView.cpp

```diff
@@ -385,14 +385,15 @@ void ListView::setDirection(Direction dir)
 {
     switch (dir)
     {
+        case Direction::NONE:
```

Comment:
In the old code, suppose we do the following:

```
listView->setDirection(Direction::NONE);
```

This will return directly at line 395. And `ScrollView::setDirection(dir);` will never be invoked. Then this function does nothing when the parameter equals to `Direction::NONE`. Though it suppose to make the listView fixed (un-scrollable)